README.md

```diff
@@ -71,57 +70,59 @@ public class Project {
     public void setScore(int score) {
         this.score = score;
     }
-
-		@Override
-    public String toString() {
-        return "Project{" +
-                "id=" + id +
-                ", title='" + title + '\'' +
-                ", submissionDate=" + submissionDate +
-                ", student=" + student +
-                ", score=" + score +
-                '}';
-    }
+    
 }
 ```
 
-Now, open the `JpaCreate` class, create two new projects, and save them in the
-database (`hibernate.hbm2ddl.auto` should be `create`):
+The directory structure should look like this:
+
+![project class added to model](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-jpa-1to1/project_structure_projectclass.png)
+
+
+1. Edit `persistence.xml` to set `hibernate.hbm2ddl.auto` to `create`.
+2. Edit the `JpaCreate` class to create three new projects  and save them in the
+database as shown below.
 
 ```java
 package org.example;
 
-import org.example.enums.StudentGroup;
-import org.example.models.IdCard;
-import org.example.models.Project;
-import org.example.models.Student;
+import org.example.model.IdCard;
+import org.example.model.Project;
+import org.example.model.Student;
+import org.example.model.StudentGroup;
 
-import javax.persistence.Persistence;
-import javax.persistence.EntityManagerFactory;
 import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
 import javax.persistence.EntityTransaction;
+import javax.persistence.Persistence;
 import java.time.LocalDate;
-import java.util.Date;
 
 public class JpaCreate {
     public static void main(String[] args) {
-        // create student instances
+        // create a new student instance
         Student student1 = new Student();
         student1.setName("Jack");
-        student1.setDob(new Date());
-        student1.setStudentGroup(StudentGroup.LOTUS);
+        student1.setDob(LocalDate.of(2000,1,1));
+        student1.setStudentGroup(StudentGroup.ROSE);
 
+        // create a new student instance
         Student student2 = new Student();
-        student2.setName("Leslie");
-        student2.setDob(new Date());
-        student2.setStudentGroup(StudentGroup.ROSE);
+        student2.setName("Lee");
+        student2.setDob(LocalDate.of(1999,1,1));
+        student2.setStudentGroup(StudentGroup.DAISY);
+
+        // create a new student instance
+        Student student3 = new Student();
+        student3.setName("Amal");
+        student3.setDob(LocalDate.of(1980,1,1));
```

Comment:
It looks like Student might still have a Date type Dob rather than LocalDate in one of the Student classes

Reply:
fixed, thanks